modules/container-insights/variables.tf

```diff
@@ -9,8 +9,6 @@ variable "helm" {
   description = "The helm release configuration"
   type        = map
   default = {
-    name            = "container-insights"
-    chart           = "container-insights"
```

Comment:
Removed default values for release name and chart name. This is because I split my local charts into 2 `eks-charts` (amazon-cloudwatch, aws-for-fluent-bit).